src/handlers/pdf-standardisation/get-standardised-invoice/get-standardised-invoice.ts

```diff
@@ -17,20 +17,32 @@ export interface StandardisedLineItem {
   due_date?: string;
   tax?: number;
   tax_payer_id?: string;
+  parser_version: string; // may not be present in old items, but required here to ensure it is added to new ones
+}
+
+export interface StandardisedLineItem extends StandardisedLineItemSummary {
   item_id?: number;
   item_description?: string;
   service_name?: string;
   unit_price?: number;
   quantity?: number;
   price?: number;
-  parser_version: string; // may not be present in old items, but required here to ensure it is added to new ones
 }
 
+export interface StandardisedLineItemFromPdfSummary
+  extends StandardisedLineItemSummary {
+  originalInvoiceFile: string; // may not be present in old items, but required here to ensure it is added to new ones
```

Comment:
This will also need to be done for `handlers/csv-extract/get-csv-standardised-invoice.ts`. We haven't processed any csvs in prod yet so there aren't any old items.